src/App.js

```diff
@@ -0,0 +1,337 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.handleRock = this.handleRock.bind(this);
+    this.handlePaper = this.handlePaper.bind(this);
+    this.handleScissors = this.handleScissors.bind(this);
+    this.computerPlay = this.computerPlay.bind(this);
+    this.game = this.game.bind(this);
+    this.show = this.show.bind(this);
+    this.handleReset = this.handleReset.bind(this);
+    this.winner = this.winner.bind(this);
+
+    this.state = {
+      computer: "",
+      human: "",
+      status: "",
+      computerCount: 0,
+      humanCount: 0,
+      totalCount: 0,
+      ties: 0,
+    }
+  }
+  computerPlay = () => {
+    const computerSelection = (Math.floor(Math.random() * 3));
+    if (computerSelection === 0) {
+      return "paper"
+    }
+    if (computerSelection === 1) {
+      return "rock"
+    }
+    if (computerSelection === 2) {
+      return "scissors"
+    }
+  }
+  handleRock() {
+    const computer = this.computerPlay()
+    const human = "rock"
+    if (human === computer) {
+      this.setState({
+        computer,
+        human,
+        totalCount: this.state.totalCount + 1,
+        ties: this.state.ties + 1
+      })
+    }
+    if (human === "scissors" && computer === "rock") {
```

Comment:
There is a lot of duplication here, these conditionals look to be the same in all three handle functions.  Can you look at DRYing it out?